src/EditorFeatures/CSharpTest/ImplementInterface/ImplementInterfaceTests.cs

```diff
@@ -8984,5 +8984,122 @@ public int Foo
 }
 ", parseOptions: CSharpParseOptions.Default.WithLanguageVersion(LanguageVersion.CSharp5));
         }
+
+        [WorkItem(53927, "https://github.com/dotnet/roslyn/issues/53927")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsImplementInterface)]
+        public async Task TestStaticAbstractInterfaceOperator_OnlyExplicitlyImplementable()
+        {
+            await TestInRegularAndScriptAsync(@"
```

Comment:
in the future, if your'e interested, it would be good to update these to be sdk style tests.  that way we can validate if our fix produces errors :)

Reply:
📝 I'm working on this change for ImplementInterfaceTests in a separate branch.